WEB/Src/DependencyCollector/Shared/Implementation/AzureSdk/AzureSdkDiagnosticsEventHandler.cs

```diff
@@ -31,30 +37,79 @@ public override void OnEvent(KeyValuePair<string, object> evnt, DiagnosticListen
                 var currentActivity = Activity.Current;
                 if (evnt.Key.EndsWith(".Start"))
                 {
-                    var telemetry = new DependencyTelemetry();
-                    
-                    SetCommonProperties(evnt.Key, evnt.Value, currentActivity, telemetry);
+                    OperationTelemetry telemetry = null;
+                    if (diagnosticListener.Name == "Azure.Core")
```

Comment:
I was trying to avoid taking dependency on source names and keep it as generic as possible.

Reply:
sure, we can use presence of http.* tag then. But this way it's a bit more efficient (and core may look better if Azure.Core will get it's own handler based on source name)

Reply:
I'm fine with both just wanted to keep it generic in case we can reuse it for convention based acitivities.